workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectDispatcherSystem.cs

```diff
@@ -143,46 +199,33 @@ protected override void OnUpdate()
                 indexManagerPair.Value.EnableSpatialOSBehaviours();
             }
 
-            foreach (var gameObjectComponentDispatcher in GameObjectComponentDispatchers)
+            foreach (var gameObjectComponentDispatcher in gameObjectComponentDispatchers)
             {
                 gameObjectComponentDispatcher.InvokeOnAuthorityLossImminentCallbacks(entityToReaderWriterStore);
                 gameObjectComponentDispatcher.InvokeOnCommandRequestCallbacks(entityToReaderWriterStore);
                 gameObjectComponentDispatcher.InvokeOnCommandResponseCallbacks(entityToReaderWriterStore);
             }
         }
 
-        public class ActivationManagerAlreadyExistsException : Exception
-        {
-            public ActivationManagerAlreadyExistsException(string message) : base(message)
-            {
-            }
-        }
-
-        public class ActivationManagerNotFoundException : Exception
-        {
-            public ActivationManagerNotFoundException(string message) : base(message)
-            {
-            }
-        }
-
-        internal void CreateActivationManagerAndReaderWriterStore(Entity entity, GameObject gameObject)
+        private void CreateActivationManagerAndReaderWriterStore(Entity entity)
         {
             if (entityToActivationManager.ContainsKey(entity))
             {
-                throw new ActivationManagerAlreadyExistsException($"MonoBehaviourActivationManager already exists for entity {entity.Index}.");
+                throw new SystemException($"MonoBehaviourActivationManager already exists for entity {entity.Index}.");
```

Comment:
Interpolate in the `nameof` the class so it matches up when things are invariably refactored/renamed (there are a few instances of this)